.jules/runs/surveyor-001/decision.md

```
@@ -0,0 +1,31 @@
+# Decision
+
+## Focus
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)